docker-compose.yaml

```diff
@@ -6,7 +6,7 @@ services:
     build:
       context: ./.config
       args:
-        grafana_version: ${GRAFANA_VERSION:-9.3.8}
```

Comment:
This should be left as a variable. Appx:
```
${GRAFANA_VERSION:10.4.0}
```

Reply:
Changed into variable.